src/msw/window.cpp

```diff
@@ -7425,7 +7421,8 @@ static TEXTMETRIC wxGetTextMetrics(const wxWindowMSW *win)
 
 #if !wxDIALOG_UNIT_COMPATIBILITY
     // and select the current font into it
-    HFONT hfont = GetHfontOf(win->GetFont());
+    wxFont f(win->GetFont());
```

Comment:
I think this change merits a comment, as it's pretty important that it doesn't get simplified back to the old version.